### PR TITLE
Personal lockers in the locker room are no longer randomized

### PR DESCRIPTION
**Pull request by Lady-Luck**
So, one thing I am guilty of, and I know I'm not the only person who does this, is opening multiple personal lockers for the bag I want. Sometimes I need a satchel, sometimes I need a duffelbag.

In other words, if I open a locker and don't find the bag I want, I will open another, thus making it so there's less personal lockers for everyone else.

This just makes all the lockers have the same bags. Before it was either one or two, now it is always the standard 3: Backpack, duffelbag, satchel.

:cl: Lady-Luck
tweak: Personal lockers in the locker room are no longer randomized and have all the same bags.
/:cl:

### CODE REVIEW

**Comment by DaveTheHeadcrab** [comment]
Personal lockers are restricted when you lock them, not when you unlock them.
So actually you aren't taking lockers from anyone.

File: code/game/objects/structures/crates_lockers/closets/secure/personal.dm [hunk 1]
```diff
@@ -33,8 +30,6 @@
 	icon_opened = "cabinetdetective_open"
 	icon_broken = "cabinetdetective_broken"
 	icon_off = "cabinetdetective_broken"
-	burn_state = FLAMMABLE
```
Comment (tigercat2000) [inline code/game/objects/structures/crates_lockers/closets/secure/personal.dm]: why did you remove these

Comment (Lady-Luck) [inline code/game/objects/structures/crates_lockers/closets/secure/personal.dm]: What the...?

**Comment by Lady-Luck** [comment]
>Personal lockers are restricted when you lock them, not when you unlock them.
So actually you aren't taking lockers from anyone.

Oh. Welp. Guess this isn't needed.

File: code/game/objects/structures/crates_lockers/closets/secure/personal.dm (hunk 1 above)
Comment (Lady-Luck) [inline code/game/objects/structures/crates_lockers/closets/secure/personal.dm]: Oh, I know what happened, I synced to the latest version of the git but I had the old file still opened, so i was editing the old file.

Moot now though.
